env/work/srw_python/srwl_uti_src.py

```diff
@@ -41,7 +35,16 @@ def srwl_uti_src_e_beam(_nm, _Iavg=None, _e=None, _sig_e=None, _emit_x=None, _be
         ['APS',                     [0.1,    7,  0.96e-03,  2.79e-09,   22.7,     0, 0.206,      0,   8.4e-12,    3.1,      0]],
         ['SPring8 High Beta',       [0.1,    8,   1.1e-03,   3.4e-09,   22.6,     0, 0.107,      0,   6.8e-12,    5.6,      0]]
     ]#add more beams
+    return allBeams
```

Comment:
Sorry for the ambiguous previous comment (I probably assumed my first record was deleted).
Can it be:
def srwl_uti_src_e_beam_predef()
and not "def srwl_uti_src_e_beams_predef" (right now it seems to be "def srwl_uti_src_e_beams_predef") ?